relayer/src/supervisor/scan.rs

```diff
@@ -322,12 +324,12 @@ impl<'a, Chain: ChainHandle> ChainScanner<'a, Chain> {
     pub fn query_allowed_channels(
         &mut self,
         chain: &Chain,
-        spec: &ChannelsSpec,
+        spec: &ChannelFilters,
```

Comment:
Should we perhaps rename this parameter from `spec` to `filters`?

Reply:
Yeah :)